tasks/simplifications.json

```diff
@@ -209,21 +209,21 @@
 		"maxmemory": "8192",
 		"skipfailures": "no"
 	},
-	"eu.geo.nuts_geometries_0_5af288f97f": {
+	"eu.geo.tasks_eu_geo_nuts_geometries_0_5af288f97f": {
```

Comment:
Maybe we want to fix this in `TableTask` instead, so the ID doesn't change? We talked about this, but I only had the hashed table names in mind, maybe the decision is different for this kind of ids? also cc @ethervoid

Reply:
I'm undoing this changes (related to https://github.com/CartoDB/bigmetadata/pull/366)